src/main/java/sx/blah/discord/api/internal/DiscordWS.java

```diff
@@ -858,11 +863,15 @@ private void voiceStateUpdate(JsonElement eventObject) {
 						client.dispatcher.dispatch(new UserVoiceChannelJoinEvent(user, channel));
```

Comment:
This block is redundant now, can you merge it with the last `else` statement? Otherwise it looks good.